packages/components/src/dropdown/index.tsx

```diff
@@ -70,8 +77,13 @@ export default function Dropdown( props ) {
 	 * case a dialog has opened, allowing focus to return when it's dismissed.
 	 */
 	function closeIfFocusOutside() {
+		if ( ! containerRef.current ) {
+			return;
```

Comment:
This is hacky, but without it there are TS errors:

<img width="928" alt="Screen Shot 2022-12-07 at 8 22 48 PM" src="https://user-images.githubusercontent.com/4063887/206340980-d7c27ea8-a57f-477c-86fe-bb79ec37af94.png">

Reply:
Thanks!